classes/admin.php

```diff
@@ -747,6 +747,12 @@ public function getPage($path)
 
                 $header = ['title' => $data['title']];
 
+                // Merge the default page headers into $header
+                $default_headers = Yaml::parse(
+                  $this->grav['config']->get('plugins.admin.frontmatter.default_headers'));
+
+                $header = (object) array_merge((array) $header, (array) $default_headers);
+
```

Comment:
IMO it seems quite unnecessary to split this particular functionality into a separate plugin, as it seems appropriate as core admin functionality, to me.

The more powerful functionality you mention is a cool idea, but doesn't need this to be in a new plugin for it to exist.

But I don't know the codebase as well as you. In case you're adamant, I'll start drafting up a plugin to do this, and raise a separate PR do this repo to create the `onAdminCreatePageFrontmatter` event (unless you particularly wanted to), and we can compare the 2 solutions I guess?